TOC.md

```diff
@@ -131,11 +132,11 @@
         + [Wrong Index Solution](/wrong-index-solution.md)
         + [Distinct Optimization](/agg-distinct-optimization.md)
       + [Prepare Execution Plan Cache](/sql-prepare-plan-cache.md)
-      + Control Execution Plan
-        + [Overview](/control-execution-plan.md)
-        + [Optimizer Hints](/optimizer-hints.md)
-        + [SQL Plan Management](/sql-plan-management.md)
-        + [The Blocklist of Optimization Rules and Expression Pushdown](/blocklist-control-plan.md)
+    + Control Execution Plans
+      + [Overview](/control-execution-plan.md)
+      + [Optimizer Hints](/optimizer-hints.md)
+      + [SQL Plan Management](/sql-plan-management.md)
+      + [Optimization Rule Blocklist and Expression Pushdown](/blocklist-control-plan.md)
```

Comment:
`Expression Pushdown` also has the `Blocklist` concept, see https://docs.pingcap.com/tidb/dev/blocklist-control-plan#the-blocklist-of-expression-pushdown

> The blocklist of expression pushdown

And the table name is `mysql.expr_pushdown_blacklist `

Reply:
I will revert this part for now. The title is a bit long though, and the problem is made slightly worse by increasing the indent level.